cloud/pkg/devicecontroller/controller/downstream.go

```diff
@@ -842,6 +833,97 @@ func deleteDeviceModelAndVisitors(deviceModel *v1alpha2.DeviceModel, deviceProfi
 	}
 }
 
+func (dc *DownstreamController) sendDeviceMsg(device *v1alpha2.Device, operation string) {
+	device.SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   "devices.kubeedge.io",
+		Version: "v1alpha2",
+		Kind:    "Device",
```

Comment:
It should use the const from api instead of the hard code

Reply:
done